ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master.yaml

```diff
@@ -43,6 +51,23 @@ tests:
     make REPORTS=/tmp/reports unit-test
   container:
     from: bin
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-assisted
+  steps:
+    cluster_profile: packet
+    dependencies:
+      ASSISTED_AGENT_IMAGE: pipeline:assisted-installer-agent
+    workflow: baremetalds-assisted
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-assisted-onprem
+  steps:
+    cluster_profile: packet
+    dependencies:
```

Comment:
Do you have to specify the dependencies here? Isn't it enough in ci-operator/step-registry/baremetalds/assisted/setup/baremetalds-assisted-setup-ref.yaml ?